src/services/textChanges.ts

```diff
@@ -442,6 +442,27 @@ namespace ts.textChanges {
             }
             throw Debug.failBadSyntaxKind(node); // We haven't handled this kind of node yet -- add it
         }
+        /**
+         * This function should be used to insert nodes alphabetically in lists when nodes don't carry separators as the part of the node range,
+         * e.g. export lists, import lists, etc.
+         * Linear search is used instead of binary as the (generally few) nodes are not guaranteed to be in order.
+         */
+        public insertNodeInListAlphabetically(sourceFile: SourceFile, containingList: NodeArray<ImportSpecifier>, newNode: ImportSpecifier): this {
+            if (newNode.name.text < containingList[0].name.text) {

```

Comment:
The way we're using this, `containingList` will contain at least one element, but it's not obvious if you were just looking at this method.
This may be too specific to belong in `textChanges` and could just go in `importFixes`. Though I think `insertNodeBefore` could be adapted so `importFixes` doesn't have to call a special `insertNodeInListBeforeFirst` method or have to call `insertNodeInListAfter` on a previous element.